### ISSUE TITLE
Using Revise before DiffEqFlux makes it fail

### ISSUE THREAD

**Issue by cqql** [timholy/Revise.jl#529]
Hi,
I have just begun integrating Revise into my workflow and there is a weird problem. If I use Revise after using DiffEqFlux everything works fine.
```
$ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.5.1 (2020-08-25)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using DiffEqFlux, Revise

julia>
```
but the other way around produces the following error.
```
$ julia
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.5.1 (2020-08-25)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> using Revise, DiffEqFlux

julia> ┌ Error: (compiled mode) evaluation error starting at /home/cqql/.julia/packages/DistributionsAD/tpspW/src/reversediff.jl:26
│   mod = DistributionsAD.ReverseDiffX
│   ex =
│    quote
│        #= /home/cqql/.julia/packages/DistributionsAD/tpspW/src/reversediff.jl:26 =#
│        const RDBroadcasted{F, T} = Broadcasted{<:Any, <:Any, F, T}
│    end
│   exception =
│    invalid redefinition of constant RDBroadcasted
│    Stacktrace:
│     [1] top-level scope at /home/cqql/.julia/packages/DistributionsAD/tpspW/src/reversediff.jl:26
└ @ Revise ~/.julia/packages/Revise/qxX5H/src/lowered.jl:85
julia>
```
Do you have any idea? Of course the workaround is easy enough but it is still peculiar.

**Comment by timholy** [comment]
Can you try the `revise3` branch? See #497 and https://discourse.julialang.org/t/ann-revise-3-0-in-beta-faster-and-better/46287

**Comment by timholy** [comment]
I don't get that warning myself on #497.

Also, if you load `DiffEqFlux` before `Revise`, then you won't be able to revise any of the code in DiffEqFlux or its dependencies.

**Comment by timholy** [comment]
Closed by #497

**Comment by cqql** [comment]
You were right, it does not happen anymore since the release. Thank you very much!
